daemon/cmd/daemon_main.go

```diff
@@ -150,6 +158,220 @@ func skipInit(basePath string) bool {
+				"tofqdns-enable-poller",
+				"tofqdns-enable-poller-events",
+				"tofqdns-endpoint-max-ip-per-hostname",
+				"tofqdns-min-ttl",
+				"tofqdns-pre-cache",
+				"tofqdns-proxy-port",
+				"tofqdns-proxy-response-max-delay",
+			},
+		},
+		{
+			name: "K8S related flags",
+			flags: []string{
+				"k8s-api-server",
+				"k8s-kubeconfig-path",
+				"k8s-namespace",
+				"k8s-require-ipv4-pod-cidr",
+				"k8s-require-ipv6-pod-cidr",
+				"k8s-watcher-endpoint-selector",
+				"k8s-watcher-queue-size",
+				"enable-k8s-event-handover",
+				"enable-k8s-external-ips",
+				"disable-k8s-services",
+				"annotate-k8s-node",
+			},
+		},
+		{
+			name: "Metrics and Monitoring related flags",
+			flags: []string{
+				"metrics",
+				"monitor-aggregation",
+				"monitor-aggregation-flags",
+				"monitor-aggregation-interval",
+				"monitor-queue-size",
+			},
+		},
+		{
+			name: "Http related flags",
+			flags: []string{
+				"http-idle-timeout",
+				"http-max-grpc-timeout",
+				"http-request-timeout",
+				"http-retry-count",
+				"http-retry-timeout",
+			},
+		},
+		{
+			name: "Debug, Logging and Trace related flags",
+			flags: []string{
+				"debug",
+				"debug-verbose",
+				"enable-tracing",
+				"log-driver",
+				"log-opt",
+				"log-system-load",
+				"envoy-log",
+			},
+		},
+		{
+			name: "IP related flags",
+			flags: []string{
+				"enable-ipv4",
+				"enable-ipv6",
+				"ip-allocation-timeout",
+				"ipam",
+				"ipsec-key-file",
+				"ipv4-cluster-cidr-mask-size",
+				"ipv4-node",
+				"ipv4-pod-subnets",
+				"ipv4-range",
+				"ipv4-service-loopback-address",
+				"ipv4-service-range",
+				"ipv6-cluster-alloc-cidr",
+				"ipv6-node",
+				"ipv6-pod-subnets",
+				"ipv6-range",
+				"ipv6-service-range",
+				"mtu",
+				"nat46-range",
+				"prometheus-serve-addr",
+			},
+		},
+		{
+			name: "KVstore related flags",
+			flags: []string{
+				"kvstore",
+				"kvstore-connectivity-timeout",
+				"kvstore-opt",
+				"kvstore-periodic-sync",
+			},
+		},
+		{
+			name: "Cluster related flags",
+			flags: []string{
+				"cluster-id",
+				"cluster-name",
+				"clustermesh-config",
+				"single-cluster-route",
+			},
+		},
+		{
+			name: "Encryption related flags",
+			flags: []string{
+				"encrypt-interface",
+				"encrypt-node",
+			},
+		},
+		{
+			name: "Endpoint related flags",
+			flags: []string{
+				"enable-endpoint-health-checking",
+				"enable-endpoint-routes",
+				"endpoint-interface-name-prefix",
+				"endpoint-queue-size",
+				"disable-endpoint-crd",
+			},
+		},
+		{
+			name: "Policy related flags",
+			flags: []string{
+				"allow-localhost",
+				"allow-icmp-frag-needed",
+				"egress-masquerade-interfaces",
+				"enable-host-reachable-services",
+				"enable-l7-proxy",
+				"enable-ipsec",
+				"enable-local-node-route",
+				"enable-policy",
+				"exclude-local-address",
+				"force-local-policy-eval-at-source",
+				"host-reachable-services-protos",
+				"install-iptables-rules",
+				"masquerade",
```

Comment:
`masquerade` and `egress-masquerade-interfaces` are more related to the datapath than to policy. Probably those + services options can go under a new section?

Reply:
Do you have section in mind.

Reply:
"Services & address translation" maybe?

Reply:
Addressed review comments, for now I just moved the masquerade to the 'others' option.